app/src/main/res/layout/view_multi_action_component.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_margin="@dimen/base_margin"
+    android:foreground="?selectableItemBackground"
+    app:cardCornerRadius="8dp">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:minHeight="88dp">
+
+        <TextView
+            android:id="@+id/title"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="@dimen/base_margin_double"
+            android:gravity="center_vertical"
+            android:textAppearance="?textAppearanceBody1"
+            android:textColor="?colorPrimary"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            tools:text="Garage" />
+
+        <TextView
+            android:id="@+id/subtitle"
+            android:layout_width="wrap_content"
+            android:layout_height="0dp"
+            android:gravity="center_vertical"
+            android:textAppearance="?textAppearanceCaption"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/title"
+            tools:text="25 sqm * water" />
+
+        <FrameLayout
```

Comment:
Should we use some type of `Button` for this? I fear that there might be some accessibility/styling implications of using a `FrameLayout`+`ImageView` 🤔

Reply:
By setting `tools:ignore="ContentDescription"` on the imageview it will not be focusable by screen readers. And by setting `style="@style/Widget.AppCompat.Button.Borderless"` on the layout it will behave like an actual button :)

Reply:
Wouldn't that make the button invisible to users using screen readers though, for example? 😞 

Could https://developer.android.com/reference/android/widget/ImageButton be appropriate for this use-case?

Reply:
Yes it would, in that case we would need to have content description on the `FrameLayout`. An `ImageButton` is an excellent idea ;)